sunpy/visualization/animator/wcs.py

```diff
@@ -281,6 +287,11 @@ def plot_start_image_2d(self, ax):
                        'origin': 'lower'}
         imshow_args.update(self.imshow_kwargs)
 
+        if self.imshow_kwargs.get('vmin') == 'auto':
+            imshow_args['vmin'], _ = self.get_2d_plot_limts()
+        if self.imshow_kwargs.get('vmax') == 'auto':
+            _, imshow_args['vmax'] = self.get_2d_plot_limts()
```

Comment:
As with https://github.com/sunpy/sunpy/pull/5025/files#r582872523, you shouldn't call your method twice.  In fact, the doubled call here and the doubled call within the method means that the underlying calculation is getting repeated *four* times.